### PR TITLE
fix: correct author alignment when date is hidden

### LINKED ISSUE #546: When the date is hidden in Content Options, the author name looks mis-aligned (issue title edited)
Can you please add a UL List that put a beautiful colored right icon to before every element.

Secondly, when we remove the date the author name looks unaligned. So when we remove the date make the author name aligned to middle in front of author gravatar.

Thanks

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR tweaks the alignment of the author's name when you've hidden the date using the theme's Content Options.

Closes #546

### How to test the changes in this Pull Request:

1. Start with the Newspack default theme.
2. Navigate to a post with multiple authors:

image

3. From there, open the Customizer > Content Options panel and uncheck 'Display date' and publish.
4. Note the appearance of the byline -- the names will sit on the high side:

image

5. Apply the PR and run `npm run build`.
6. Confirm that the names are now nicely aligned:

image

8. Cycle through the other themes; Newspack Sacha in particular needed the styles tweaked for smaller screens, as well as the featured image behind and beside styles, to make sure the byline is still centred:

image

image

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: newspack-sacha/sass/style.scss [hunk 1]
```diff
@@ -258,6 +258,30 @@ cite {
 		}
 	}
 
+	@include media( mobileonly ) {
+		&.date-hidden .entry-header .entry-meta {
+			display: block;
+		}
+	}
+
+	@include media( tablet ) {
```
Comment (adekbadek) [inline newspack-sacha/sass/style.scss:267]: Is this block (lines 267-275) necessary? This media query contains the same CSS as below, so it seems to defeat the purpose of the media query.

File: newspack-sacha/sass/style.scss [hunk 2]
```diff
@@ -258,6 +258,30 @@ cite {
...
+		}
+	}
+
+	&.date-hidden .featured-image-behind .entry-meta {
+		display: block;
+
+		.byline {
+			margin: 0;
+		}
+	}
```
[the remark this answers is not in the document]
Comment (laurelfulford) [reply newspack-sacha/sass/style.scss:285]: Ah, good call -- thanks Adam! This has been tidied up in [bbc80c1](https://github.com/Automattic/newspack-theme/pull/1777/commits/bbc80c1c651f11318f47564416496b641e7020fb)!

File: newspack-sacha/sass/style.scss (hunk 1 above)
Comment (laurelfulford) [reply newspack-sacha/sass/style.scss]: The classnames are _near_ identical (seriously, like three characters apart!), but they're actually sliiiightly different styles for the featured image behind and beside styles. For the 'beside' ones, the layout switches to "normal" for tablet-sized screens (with the author to the left and social links to the right), and the the 'behind' ones stay the same on all screen sizes. 

Just let me know if this doesn't make sense!

Comment (adekbadek) [reply newspack-sacha/sass/style.scss]: Oh, my bad! I was not parsing all the letters in these class names 🤦 :D

**Review by adekbadek — APPROVED** [review]

File: newspack-sacha/sass/style.scss (hunk 1 above)
Comment (laurelfulford) [reply newspack-sacha/sass/style.scss]: No worries! 😄 I seriously could not have come up with worse class names if I tried!!

**Comment by laurelfulford** [comment]
Thanks @adekbadek!

**Pull request merged**